cloudinary-core/src/main/java/com/cloudinary/Util.java

```diff
@@ -226,4 +228,36 @@ private static void putArray(String name, Map from, Map<String, Object> to) {
     protected static String timestamp() {
         return Long.toString(System.currentTimeMillis() / 1000L);
     }
+
+    public static byte[] getUTF8Bytes(String string) {
```

Comment:
Since this was made public we need documentation. Same goes for `produceSignature` below.
Please also document all new methods/classes (`ApiResponseSignatureVerifier`, `NotificationRequestSignatureVerifier`, etc).